programs/server/play.html

```diff
@@ -119,17 +127,21 @@
             background-color: var(--element-background-color);
         }
 
-        #query
+        #query_div
         {
             /* Make enough space for even huge queries. */
             height: 20%;
+        }
+
+        #query
+        {
+            height: 100%;
             width: 100%;
```

Comment:
Maybe add: `resize: vertical;` here - I don't think you need should be able to resize horizontally